src/services/docGenService.ts

```diff
@@ -402,6 +416,8 @@ export class DocGenService {
                   message.user_instructions.prompt_hint || "generate",
               },
               follow_up_instructions: message.follow_up_instructions,
+              column_index_count: columnIndexCount,
```

Comment:
@saravmajestic When retry happens due to rate limiting, the columnIndexCount does not change right?

Reply:
it wont change. it will pickup same index